modules/vulnerability-management20-configure-report-delivery-destinations-schedule.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-vulnerabilities.adoc
+:_content-type: PROCEDURE
+[id="vulnerability-management20-configure-report-delivery-destinations-schedule_{context}"]
+= Configuring delivery destinations and scheduling
+
+[role="_abstract"]
+You can optionally configure destinations for vulnerability reports and set up a schedule for delivery.
+
+.Procedure
+. To configure destinations for delivery, in the *Create report* window, under *Configure delivery destinations*, click *Add delivery destination*.
+. To email reports, you must configure at least one email notifier. Select an existing notifier or create a new email notifier to send your report by email. For more information on creating an email notifier, see "Configuring the email plugin" in the "Additional resources" section.
+. When you select a notifier, the email address associated with that notifier appears in the *Distribution list* field. You can add additional email addresses that are separated by a comma.
```

Comment:
There are a couple of email fields associated with an email notifier. Therefore, this must be made clearer by indicating specific which email addresses are being talked about.  The email addresses configured as `Default recipients` only are auto-shown in notifiers.